scripts/install.sh

```diff
@@ -7,7 +7,17 @@ INSTALL_DIR=/usr/local/bin/
 BINARY=xq
 LATEST_VERSION=$(curl -L -s -H 'Accept: application/json' $URL_PREFIX/releases/latest | sed -e 's/.*"tag_name":"v\([^"]*\)".*/\1/')
 PLATFORM=$(uname -s | tr A-Z a-z)
-[ "$(uname -m)" = "x86_64" ] && ARCH=amd64 || ARCH=arm64
+case "$(uname -m)" in
+  x86_64)
+    ARCH=amd64
+    ;;
+  armv6l)
+    ARCH=armv6
+    ;;
+  *)
```

Comment:
Despite the original "if" statement, the default fallback case probably should be "amd64".